src/modules/LanUdevAuto.rb

```diff
@@ -194,39 +198,34 @@ def Import(settings)
     end
 
     def Write
-      comment = [
-        "# Generated by autoyast",
-        "# program run by the persistent-net-generator.rules rules file.",
-        "#",
-        "# You can modify it, as long as you keep each rule on a single line."
-      ]
+      template = "SUBSYSTEM==\"net\", ACTION==\"add\", DRIVERS==\"?*\", %s==\"%s\", NAME=\"%s\""
```

Comment:
for longer template I really recomment to use named replacement points. It really helps with applying and also with reading code like:
```
template = "SUBSYSTEM==\"net\", ACTION==\"add\", DRIVERS==\"?*\", %{rule}==\"%{rule_value}\", NAME=\"%{device_name}\""

# and then usage
rules << template % [ rule: opt, rule_value: value.downcase, device_name: devname]
```